VirtoCommerce.Storefront/Controllers/Api/ApiListsController.cs

```diff
@@ -99,6 +99,43 @@ public async Task<ActionResult<ShoppingCartItems>> RemoveItemFromList(string lin
             }
         }
 
+        // PUT: storefrontapi/lists/{listName}/{type}/items/{lineItemId}?quantity=...
+        [HttpPut("{listName}/{type}/items/{lineItemId}")]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> ChangeListItem(string listName, string type, string lineItemId, int quantity)
```

Comment:
Instead of pass lineItemId and quantity as parameters better use https://github.com/VirtoCommerce/vc-storefront-core/blob/dev/VirtoCommerce.Storefront.Model/Cart/ChangeCartItemQty.cs type. and don't forget to add binding attributes to al parameters [FromRoute] and [FromBody]